src/prx/main.py

```diff
@@ -231,13 +228,14 @@ def build_records(
     flat_obs.obs_value = flat_obs.obs_value.astype(float)
     flat_obs[["sv", "obs_type"]] = flat_obs[["sv", "obs_type"]].astype(str)
 
-    flat_obs = flat_obs.rename(
+    flat_obs.rename(
         columns={
             "time": "time_of_reception_in_receiver_time",
             "sv": "satellite",
             "obs_value": "observation_value",
             "obs_type": "observation_type",
         },
+        inplace=True,
```

Comment:
I'm a little divided on `inplace=True`, as there are [claims](https://towardsdatascience.com/why-you-should-probably-never-use-pandas-inplace-true-9f9f211849e4) that in many cases copies are generated amyways, therby negating the expected performance benefit. Did you notice a speedup with `inpplace=True`? If not, I'd vote in favor of not using it so that the default expectation in `prx` code can always be that DataFrame operations return something.

Reply:
Interesting blog post!
Initially, I did this particular modifications because things were not working with `dask` for delayed/lazy computations. I guess the tasker had difficulty to track the renaming of the columns before doing the job. Since I do not think that I will manage to use `dask`, we can remove it.

I tested with and without `inplace=True` and there was no difference. I committed the removal of it in c71b75e.

Things are soo faaast now, well done!